core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestPushProjectionIntoTableScan.java

```diff
@@ -199,12 +199,6 @@ public void testPushProjection()
                                             .collect(toImmutableMap(Map.Entry::getKey, e -> e.getValue()::equals)),
                                     Optional.of(PlanNodeStatsEstimate.builder()
                                             .setOutputRowCount(42)
-                                            .addSymbolStatistics(new Symbol(newNames.get(constant)), SymbolStatsEstimate.builder()
-                                                    .setDistinctValuesCount(1)
-                                                    .setNullsFraction(0)
-                                                    .setLowValue(5)
-                                                    .setHighValue(5)
```

Comment:
why removed?

Reply:
Symbol statistics for a constant did not make any sense.

Reply:
does it mean we no longer have statistics for the symbol that happens to be a constant projection?

Reply:
Yes. Since constant projections will be filtered out before reaching the connector, statistics for constant symbols won't make any sense. WDYT ?

Reply:
This is not stats provided to the connector.
This is stats provided to the engine (the optimizer).
After the change, the optimizer no longer knows that the symbol representing a constant is actually representing a constant.

Please restore this assertion.
The code responsible for producing this value needs to be updated. This is here
https://github.com/trinodb/trino/blob/768ff6f63a2f94e017c9c27f0c8b0c191b05425e/core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/PushProjectionIntoTableScan.java#L180-L196